notebooks/tutorial_ivf_pq.ipynb

```diff
@@ -0,0 +1,1226 @@
+{
```

Comment:
<blockquote>Besides <code>n_probes</code>, <code>ivf_pq.SearchParams</code> contains couple more parameters, which affect internal workings of the algorithm. </blockquote><p>Besides <code>n_probes</code>, <code>ivf_pq.SearchParams</code> contains a couple more parameters, which affect the internal workings of the algorithm. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/raft/pull/1544/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>